app/helpers/routes/getOrderItemPageData.test.js

```diff
@@ -75,13 +75,20 @@ describe('getOrderItemPageData', () => {
     });
 
     it('should return the formData with the price from getSelectedPrice', async () => {
```

Comment:
Does the test description need changing? (I can't see any reference to `getSelectedPrice` in the test.)